ntbs-service/wwwroot/source/Components/FetchSpecimenPotentialMatch.ts

```diff
@@ -29,11 +29,12 @@ const FetchSpecimenPotentialMatch = Vue.extend({
 
             axios.request(requestConfig)
                 .then((response: any) => {
-                    if (typeof response.data === "object") {
-                        this.showErrorMessage(response.data.errorMessage);
-                    } else {
+                    if (typeof response.data !== "object") {
                         this.updatePotentialMatchHtml(response.data);
+                    } else if (response.data.notificationIdSpecified === true) {
+                        this.showErrorMessage(response.data.errorMessage);
                     }
```

Comment:
I could have inserted the empty/whitespace check here, but it felt better to consolidate all of the validation into one method (in the C# backend)

Reply:
I don't think we need to make a change here - I think we can just return an empty OK result, and then this component will interpret that as emptying the potential match field, which is what we want.

Reply:
Yeah sure, I've changed that over